src/ontology/edits/oeo-physical.omn

```diff
@@ -3315,10 +3315,16 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/952",
 Class: OEO_00010000
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Ammonia is a portion of matter with the chemical formula NH3. It has a gaseous normal state of matter and can act as an air pollutant. Synonyms are trihydridonitrogen and nitrogen trihydride.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Ammonia is a portion of matter with the chemical formula NH3. It has a gaseous normal state of matter and can act as an air pollutant. As it can be oxidised it can be used as a fuel.",
         <http://purl.obolibrary.org/obo/IAO_0000118> "NH3",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "nitrogen trihydride",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "trihydridonitrogen",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/263
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/356",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/356
+
+Adapt definiton and axioms, add alternative  terms:
+https://github.com/OpenEnergyPlatform/ontology/issues/939
+https://github.com/OpenEnergyPlatform/ontology/pull/959",
```

Comment:
'term tracker item' is incorrect. Description of the two lines are missing for all terms.